include/odp/api/spec/ipsec.h

```diff
@@ -491,11 +491,11 @@ typedef struct odp_ipsec_ipv6_param_t {
  * pointers and copied byte-by-byte from memory to the packet.
  */
 typedef struct odp_ipsec_tunnel_param_t {
-	/** Tunnel type: IPv4 or IPv6 */
+	/** Tunnel type: IPv4 or IPv6. The default is IPv4. */
 	odp_ipsec_tunnel_type_t type;
 
-	/** Variant mappings for tunnel parameters */
-	union {
+	/** Tunnel type specific parameters */
+	struct {
```

Comment:
Missed this earlier. Is this change required? Or are we replacing all unions with struct in general? We are not specifying default values for these, right?

Reply:
Defaults are defined at odp_ipsec_ipv4_param_t  and odp_ipsec_ipv6_param_t definition e.g. ttl and hlimit.

Reply:
Missed it. Thanks!